Composer/packages/server/schemas/sdk.uischema

```diff
@@ -145,6 +145,18 @@
       "subtitle": "Delete Property"
     }
   },
+  "Microsoft.DeleteActivity": {
+    "form": {
+      "label": "Delete activity"
+    }
+  },
```

Comment:
Let's make sure this gets added to the SDK component schemas.

Reply:
https://github.com/microsoft/botbuilder-dotnet/issues/4514